HSTracker/Database/RealmHelper.swift

```diff
@@ -328,7 +328,7 @@ struct RealmHelper {
         do {
             try realm.write {
                 if update {
-                    realm.add(deck, update: update)
+                    realm.add(deck, update: .all)
```

Comment:
Is that done on purpose?

Reply:
Just to clarify the reasoning here, I recall it was a deprecated method and when I looked up the old code implementation, it did the equivalent of the above.